packages/css-vars/src/index.ts

```diff
@@ -1,14 +1,12 @@
-let counter = -1;
-
-const unique = <Name extends string>(name: Name) => {
-  return `${name}-${++counter}` as const;
-};
-
-const define = <Name extends string>(name: Name, unique = false) => {
-  if (unique) {
-    return `--${name}` as const;
+const WEBSTUDIO_CSS_VARIABLE_POSTFIX = "-webstudio-variable";
```

Comment:
css-vars package was so far generic, not webstudio specific, I would keep it that way as long as possible, one way is to just use genearted hash, or maybe some generic suffix or define the suffix from application layer ....

Reply:
By package name
```
-webstudio-css-vars
```